seatunnel-e2e/seatunnel-flink-connector-v2-e2e/src/test/java/org/apache/seatunnel/e2e/flink/v2/jdbc/JdbcPhoenixIT.java

```diff
@@ -0,0 +1,152 @@
+
+@Slf4j
+public class JdbcPhoenixIT extends FlinkContainer {
+
+    private static final String PHOENIX_DOCKER_IMAGE = "iteblog/hbase-phoenix-docker:1.0";
+
+    private static final String PHOENIX_CONTAINER_HOST = "flink_e2e_phoenix_sink";
+    private static final String PHOENIX_HOST = "localhost";
+
+    private static final int PHOENIX_PORT = 8764;
+    private static final int PHOENIX_CONTAINER_PORT = 8765;
+
+    private static final String PHOENIX_CONNECT_URL = String.format("jdbc:phoenix:thin:url=http://%s:%s;serialization=PROTOBUF", PHOENIX_HOST, PHOENIX_PORT);
+    private static final String PHOENIX_JDBC_DRIVER = "org.apache.phoenix.queryserver.client.Driver";
+
+    private GenericContainer<?> phoenixServer;
+
+    private Connection connection;
+
+    @BeforeEach
+    public void startPhoenixContainer() throws ClassNotFoundException, SQLException {
+        phoenixServer = new GenericContainer<>(PHOENIX_DOCKER_IMAGE)
+                .withNetwork(NETWORK)
+                .withNetworkAliases(PHOENIX_CONTAINER_HOST)
+                .withLogConsumer(new Slf4jLogConsumer(log));
+        phoenixServer.setPortBindings(Lists.newArrayList(
+                String.format("%s:%s", PHOENIX_PORT, PHOENIX_CONTAINER_PORT)));
+        Startables.deepStart(Stream.of(phoenixServer)).join();
+        initializeJdbcConnection();
+        log.info("phoenix container started");
+        initializePhoenixTable();
+        batchInsertData();
+    }
+
+    @Test
+    public void testJdbcPhoenixSourceAndSink() throws IOException, InterruptedException, SQLException {
+        Container.ExecResult execResult = executeSeaTunnelFlinkJob("/jdbc/jdbc_phoenix_source_and_sink.conf");
+        Assertions.assertEquals(0, execResult.getExitCode());
+
+        // query result
+        String sql = "select age, name from test.sink order by age asc";
+        List<List> result = new ArrayList<>();
+        try (Statement statement = connection.createStatement()) {
+            ResultSet resultSet = statement.executeQuery(sql);
+            while (resultSet.next()) {
+                result.add(Arrays.asList(
+                        resultSet.getInt(1),
+                        resultSet.getString(2)));
+            }
+        }
+        Assertions.assertIterableEquals(generateTestDataset(), result);
+    }
+
+    private void initializeJdbcConnection() throws SQLException, ClassNotFoundException {
+        Class.forName(PHOENIX_JDBC_DRIVER);
+        connection = DriverManager.getConnection(PHOENIX_CONNECT_URL);
+    }
+
+    private void initializePhoenixTable() {
+        try  {
+            Statement statement = connection.createStatement();
+            String createSource = "create table test.source(\n" +
+                    " name VARCHAR PRIMARY KEY,\n" +
+                    " age INTEGER)";
+            String createSink = "create table test.sink(\n" +
+                    " name VARCHAR PRIMARY KEY,\n" +
+                    " age INTEGER)";
```

Comment:
Can you test more data types? reference to PhoenixTypeMapper

Reply:
> Can you test more data types? reference to PhoenixTypeMapper

added more phoenix data types in testcase and these types different from MySQL